prep-css-box-model/styles.css

```diff
@@ -0,0 +1,19 @@
+/* stylelint-disable no-duplicate-selectors */
+body {
+  text-align: center;
+}
+
+button {
+  background-color: red;
+  color: white;
+
+  /* 👇 YOUR CODE HERE 👇 */
+}
+
+button {
+  margin: 20px;
+  padding: 10px;
+  border: 3px;
+  border-style: double;
+  border-color: darkgray;
```

Comment:
This code should be in the original button selector, there is no reason to add another